tree/ntuple/v7/inc/ROOT/RMiniFile.hxx

```diff
@@ -48,15 +50,26 @@ Only the RNTuple key will be listed in the list of keys. Like TBaskets, the page
 Byte offset references in the RNTuple header and footer reference directly the data part of page records,
 skipping the TFile key part.
 
-While the class is central to anchoring an RNTuple in a TFile, it is an internal detail not exposed to users.
-Note that there is no user-facing RNTuple class but RNTupleReader and RNTupleWriter.
+In the list of keys, this object appears as "ROOT::Experimental::RNTuple".
+
+The RNTuple object is the user-facing representation of an RNTuple data set in a ROOT file.
+The RFileNTupleAnchor is the low-level entry point of an RNTuple in a ROOT file used by the page storage layer.
+
+The ROOT::Experimental::RNTuple object has the same on-disk layout as the RFileNTupleAnchor.
+It only adds methods and transient members. Reading and writing RNTuple anchors with TFile and the minifile writer
+is thus fully interoperable (same on-disk information).
+TODO(jblomer): Remove unneeded fChecksum, fVersion, fSize, fReserved once ROOT::Experimetnal::RNTuple moves out of
+the experimental namespace.
 */
 // clang-format on
-struct RNTuple {
+struct RFileNTupleAnchor {
+   /// The ROOT streamer info checksum. Older RNTuple versions used class version 0 and a serialized checksum,
+   /// now we use class version 3 and "promote" the checksum as a class member
+   std::int32_t fChecksum = 0;
```

Comment:
I guess that these changes break backward compatibility to read files generated prior to this PR.  Therefore, after merging:

TODO: we might need to regenerate some files used for tests in `rootbench` (IIRC, nothing in `roottest` though).

Reply:
It's the reverse, the `fChecksum` parameter is now part of the struct to retain backward compatibility with already generated files.  Once we rename `ROOT::Experimental::RNTuple` to `ROOT::RNTuple`, we will break backwards compatibility anyway and then is a good moment to clean up the data members.